aldryn_search/base.py

```diff
@@ -145,3 +153,39 @@ def prepare_fields(self, obj, language, request):
             self.prepared_data['text'] = clean_join(
                 ' ', [prepared_title, prepared_text]
             )
+
+
+class AldrynAppconfigIndexBase(AldrynIndexBase):
+
+    """
+    This class add a check for objects that have app config instances related
+    to it.
+    """
+
+    def check_for_app_config(self, obj):
+        """
+        This should check for app_config in object and if it exists check
+        if app_config namespace exists in some page. If app_config exists
+        and namespace does not exists in pages we should skip indexing for
+        this object. If app_config does not exists for given object or we
+        can't import appconfig stuff, just ignore.
+        """
```

Comment:
Very minor point, but heredocs should be a lot more declarative. This is a very nice comment though and I would love to see everyone us them more =)